static/app/views/dashboards/widgetCard/index.tsx

```diff
@@ -511,6 +512,15 @@ export const WidgetCardPanel = styled(Panel, {
   min-height: 96px;
   display: flex;
   flex-direction: column;
+
+  :hover
+    ${StyledWidgetCardContextMenuContainer},
+    :focus-within
+    ${StyledWidgetCardContextMenuContainer} {
+    width: auto;
+    opacity: 1;
+    transform: scale(1);
```

Comment:
We're using a combination of this to control visibility so we can preserve tab accessibility. Not sure if there's a better way, but this seems reasonable.

Reply:
Agreed that this is reasonable, but I also found this pattern in the codebase, which looks great: https://github.com/getsentry/sentry/blob/6b88b974fcd116fcb68d382c1253434520e541b0/static/app/components/searchQueryBuilder/tokens/deletableToken.tsx#L145-L153

maybe worth extracting to somewhere? Or at a minimum, stealing it! Discussed in detail here: https://css-tricks.com/inclusively-hidden/

Reply:
Nice find!